ament_cmake_python is missing the dependency to ament_cmake_core

The following lines without a `find_package(ament_cmake_core)`:
```cmake
find_package(ament_cmake_python)
ament_get_python_install_dir (PYTHON_SITE_PACKAGES)
```
result in this error:
```
CMake Error at /opt/ros/jazzy/share/ament_cmake_python/cmake/ament_cmake_python-extras.cmake:59 (get_executable_path):
  Unknown CMake command "get_executable_path".
Call Stack (most recent call first):
  /opt/ros/jazzy/share/ament_cmake_python/cmake/ament_get_python_install_dir.cmake:23 (_ament_cmake_python_get_python_install_dir)
  CMakeLists.txt:172 (ament_get_python_install_dir)
```
Obviously, `ament_cmake_python` is missing the `find_package(ament_cmake_core)` call, which declares the command `get_executable_path`.

--- a/ament_cmake_python/ament_cmake_python-extras.cmake
+++ b/ament_cmake_python/ament_cmake_python-extras.cmake
@@ -56,6 +56,7 @@ elif 'rpm_prefix' in schemes:
     kwargs['scheme'] = 'rpm_prefix'
 print(os.path.relpath(sysconfig.get_path('purelib', **kwargs), start='${CMAKE_INSTALL_PREFIX}').replace(os.sep, '/'))"
     )
+    find_package(ament_cmake_core QUIET REQUIRED)
     get_executable_path(_python_interpreter Python3::Interpreter CONFIGURE)
     execute_process(
       COMMAND

